Dockerfile.test.gpu

```diff
@@ -97,7 +97,7 @@ RUN pip install "Pillow<7.0" --no-deps
 
 # Install MXNet.
 RUN if [[ ${MXNET_PACKAGE} == "mxnet-nightly" && ${PYTHON_VERSION} != "2.7" ]]; then \
-        pip install --pre mxnet-cu101 -f https://dist.mxnet.io/python/all; \
+        pip install --pre mxnet-cu101mkl -f https://dist.mxnet.io/python/all; \
```

Comment:
@apeforest Horovod with MXNet nightly build is failing with:
```
In file included from horovod/mxnet/mpi_ops.h:23:0,
                 from horovod/mxnet/mpi_ops.cc:20:
/usr/local/lib/python3.6/dist-packages/mxnet/include/mxnet/ndarray.h:41:10: fatal error: mkldnn.hpp: No such file or directory
```
I don't know if this change above will fix it. Any idea?

Reply:
Ok, that fixed!